api/src/main/resources/liquibase.xml

```diff
@@ -195,4 +195,11 @@
 			DELETE FROM person_attribute WHERE value IS NULL OR value = '';
 		</sql>
 	</changeSet>
+	<changeSet id="ugandaemr-01-10-2018-1200" author="slubwama">
+		<comment>Move all Patients Ever enrolled in Facility Based Individual Management DSDM </comment>
+		<sql>
+			INSERT INTO patient_program (patient_id, program_id, date_enrolled, location_id, creator, date_created, voided, uuid)
+			select DISTINCT patient_id,5,encounter_datetime as date_enrolled,location_id,creator,date_created,0,uuid() as uuid from encounter where encounter_type=8;
```

Comment:
@slubwama 

1. Use SELECT with a GROUP BY patient_id as that will return only one value per patient_id

2. Use the encounter_type uuid instead of the id since these values may change in some installations to be on the safe side

Reply:
@ssmusoke the group by patient will end up having multiple FBIM at the first time yet its not our desirable output

Reply:
@slubwama 

1. you join to the encounter_type table please note that this is run only once

2. I have just run a query on the distinct and I have duplicate patient ids returned. Group by will always return one row and no more even when duplicates exist